src/containers/Home.js

```diff
@@ -39,6 +52,13 @@ class Home extends Component {
       <div>
         <p>Welcome to Pocket IMDb</p>
         <h4>Movies</h4>
+        <Search
+          perPage={this.props.perPage}
+          pageNumber={this.props.activePage}
+          fetchMovies={(title, pageNumber, elementsPerPage) =>
```

Comment:
Move this to a function inside component, because this way it will create a new function on every render
```javascript
fetchMovies={this.handleFetchMovies}
```